docs/querying/sql-operators.md

```diff
@@ -37,6 +37,10 @@ sidebar_label: "Operators"
 
 Operators in [Druid SQL](./sql.md) typically operate on one or two values and return a result based on the values. Types of operators in Druid SQL include arithmetic, comparison, logical, and more, as described here. 
 
+When performing math operations, Druid uses the integer datatype unless there are double or float values. If double or float values are involved, Druid uses double. Note that the highest precision way to store digits in Druid are 64-bit integers (long) or 64-bit floats (double). In essence, a double can represent 52 binary digits, so Druid may return incorrect results for doubles if the value exceeds 2^52.
```

Comment:
Reword: Druid uses the 64-bit integer (long) data type...

> If double or float values are involved, Druid uses double.

Not sure what this means. Is it:

* If an operation uses float or double values, then the result is double.
* If an operation uses only float types, the result type is float. If an operation uses only double values, or both double and float values, the result is double.

Nit: "datatype" is not really a word. Consider "data type". 

Probably worth saying that the precision of float and double are defined by Java and by the IEEE standard. Perhaps we can link to those reference materials.